appveyor.yml

```diff
@@ -40,8 +40,10 @@ platform:
     - x64
 
 install:
+    # We don't need to build the example recipe.
+    - cmd: rmdir recipes\example /s /q
     # Skip if we are not in a PR.
-    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER exit 0
+    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER del /f /q recipes\*
```

Comment:
This doesn't appear to work. Instead try the following.

    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER rmdir /q /s recipes
    - cmd: if not exist recipes mkdir recipes